src/lib/modules/Series/SeriesCard.js

```diff
@@ -42,12 +42,10 @@ export class SeriesCard extends Component {
       : null;
     return (
       <Card
-        link
         centered
         className="fs-book-card"
-        onClick={() =>
-          goTo(FrontSiteRoutes.seriesDetailsFor(data.metadata.pid))
-        }
+        as={Link}
```

Comment:
so apparently this fixes the opening in new tab?

Reply:
this was only the old code leftover, the link way is cleaner, the actual fix is in the `DocumentCard`